static-assets/components/cstudio-forms/controls/file-name.js

```diff
@@ -116,10 +116,10 @@ YAHOO.extend(CStudioForms.Controls.FileName, CStudioForms.CStudioFormField, {
   },
 
   getRequirementCount: function () {
```

Comment:
There are two possible validations (requirements): required and path available. Since they can't collide it can be treated as just one requirement (already done in other controls)

Reply:
Yeah at the same level, higher in the code it doesn't handle it as one, just adds the count to the total display at the top of the form section